ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/ccj/CCDCountyCourtJudgment.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.cmc.ccd.domain.ccj;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentSchedule;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Value
+@Builder
+public class CCDCountyCourtJudgment {
+
+    private LocalDate defendantDateOfBirth;
+    private BigDecimal paidAmount;
+    private CCDPaymentOption paymentOption;
+    private BigDecimal repaymentPlanInstalmentAmount;
+    private LocalDate repaymentPlanFirstPaymentDate;
+    private LocalDate repaymentPlanCompletionDate;
+    private String repaymentPlanPaymentLength;
+    private CCDPaymentSchedule repaymentPlanPaymentSchedule;
+    private LocalDate payBySetDate;
+    private String statementOfTruthSignerName;
+    private String statementOfTruthSignerRole;
+    private CCDCountyCourtJudgmentType ccjType;
```

Comment:
This needs changing to `type` as agreed during our latest conversation but let's do it in a separate PR.